spec/kitchen/driver/openstack_spec.rb

```diff
@@ -211,28 +212,6 @@
       end
     end
 
-    context 'when executed with a bourne shell' do
```

Comment:
Did we lose these tests? I see the non-bourne shell tests got added below, but not these.